src/java.desktop/share/classes/sun/print/RasterPrinterJob.java

```diff
@@ -2092,7 +2092,7 @@ protected int getNoncollatedCopies() {
     private AffineTransform defaultDeviceTransform;
     private PrinterGraphicsConfig pgConfig;
 
-    synchronized void setGraphicsConfigInfo(AffineTransform at,
+    protected synchronized void setGraphicsConfigInfo(AffineTransform at,
                                             double pw, double ph) {
```

Comment:
```suggestion
    protected synchronized void setGraphicsConfigInfo(AffineTransform at,
                                                      double pw, double ph) {
```
To preserve alignment.